src/doc/data_into_visit/SiloFormat.rst

```diff
@@ -1,6 +1,1621 @@
+Silo_ is not distributed as part of the VisIt_ source code or binary installations so you must obtain it separately unless you are developing on the Windows platform.
+A link to the most up-to-date version of the Silo_ library's source code can be found on `Silo's Github site <https://github.com/LLNL/Silo/releases>`_.
+
+Once you download the Silo_ source code, building and installing it is usually only a matter of running its *configure* script and running *make*.
+You can even use the build_visit script from the `VisIt Web site <https://github.com/visit-dav/visit/releases>`_ to build Silo_ with support for HDF5.
+An example command line to build Silo_ with support for HDF5 is:
+
+.. code-block:: bash
+
+  ./build_visit --console --no-visit --no-thirdparty \
+  --thirdparty-path /usr/local \
+  --silo --hdf5 --szip
+
+After you've configured, built, and installed the Silo_ library, your program will have to be built against the Silo_ library.
+Building against the Silo_ library is usually accomplished by a simple adaptation of your *Makefile* and the inclusion of *silo.h* in your C-language source code.
+If you used build_visit to install Silo_ and its dependant libraries in /usr/local then you would add the following to your Makefile, adjusting the values for the install location, versions, and PLATFORM accordingly:
+
+.. code-block::
+
+  PLATFORM=i386-apple-darwin10_gcc-4.2
+  SZIP_DIR=/usr/local/szip/2.1/$(PLATFORM)
+  SZIP_CPPFLAGS=-I$(SZIP_DIR)/include
+  SZIP_LDFLAGS=-L$(SZIP_DIR)/lib
+  SZIP_LIBS=-lsz
+  HDF5_DIR=/usr/local/hdf5/1.8.4/$(PLATFORM)
+  HDF5_CPPFLAGS=-I$(HDF5_DIR)/include $(SZIP_CPPFLAGS)
+  HDF5_LDFLAGS=-L$(HDF5_DIR)/lib $(SZIP_LDFLAGS)
+  HDF5_LIBS=-lhdf5 $(SZIP_LIBS) -lz
+  SILO_DIR=/usr/local/silo/4.6.2/$(PLATFORM)
+  SILO_CPPFLAGS=-I$(SILO_DIR)/include $(HDF5_CPPFLAGS)
+  SILO_LDFLAGS=-L$(SILO_DIR)/lib $(HDF5_LDFLAGS)
+  SILO_LIBS=-lsiloh5 $(HDF5_LIBS) -lm
+  LDFLAGS=$(LDFLAGS) $(SILO_LDFLAGS)
+  LIBS=$(SILO_LIBS)
+  CPPFLAGS=$(CPPFLAGS) $(SILO_CPPFLAGS)
+
+If your *Makefile* does not use *CPPFLAGS* then you might try adding the `-I` include directives to *CFLAGS*, *F77FLAGS*, or whichever make variables are relevant for your Makefile.
+
+Using Silo on Windows
+"""""""""""""""""""""
+
+When you build an application using the Silo_ library on Windows, you can use the precompiled Silo_ DLL and import library that comes with the VisIt_ development distribution for Windows: *visit_windowsdev_x.y.x.zip*, where *x.y.z* refers to the version, like *3.3.3*.
+The development distribution for Windows includes pre-built binaries (.dlls and import libraries) for the Third party libraries upon which VisIt_ depends, including Silo_.
+Simply unzip the distribution to whichever location best suits your needs.
+The binaries are located in the *windowsbuild/MSVC<VERSION>* folder, with *<VERSION>* being the version of Visual Studio they were built with (eg MSVC2017).
```

Comment:
Does this section on Windows reflect the *current* way to build and use Silo on windows?

Reply:
This section only talks about how to locate prebuilt silo provided by visit for use in a private build of some other app.

It does not address building Silo itself of Windows.

Reply:
Ok, that is fine I think for now. Maybe add another ticket to address this.